Cargo.toml

```diff
@@ -1,21 +1,21 @@
 [workspace]
-default = "cycles-ledger"
+resolver = "2"
 members = [
-    # Production crates
-    "cycles-ledger",
+  # Production crates
+  "cycles-ledger",
 
-    # Crates used for testing only
-    "depositor",
-    "fake-cmc"
+  # Crates used for testing only
+  "depositor",
+  "fake-cmc",
 ]
 
 [workspace.package]
-version = "0.2.1"
+version = "0.3.0"
 edition = "2021"
 authors = ["DFINITY Stiftung <sdk@dfinity.org>"]
 license = "Apache-2.0"
 repository = "https://github.com/dfinity/cycles-ledger"
-publish = false # don't publish to crates.io
+publish = false                                         # don't publish to crates.io
```

Comment:
What's the idea behind this big space?

Reply:
The VSCode "Even Better TOML" extension auto-format it.